From b606c6afe84e9f50ae0877249e1d797c1230651b Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Thu, 6 Oct 2011 17:34:04 +0100
Subject: [PATCH] GAction API changed in 2.29.90

(or earlier, but it's the one we're trying to compile against)
---
 mx/mx-action.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mx/mx-action.c b/mx/mx-action.c
index 0faf5eb..9568a31 100644
--- a/mx/mx-action.c
+++ b/mx/mx-action.c
@@ -204,8 +204,8 @@ g_action_iface_init (GActionInterface *iface)
   iface->get_state_hint = mx_action_get_state_hint;
   iface->get_enabled = mx_action_get_enabled;
   iface->get_state = mx_action_get_state;
-  /* the set_state virtual function was renamed to change_state in glib 2.30 */
-#if GLIB_CHECK_VERSION (2, 30, 0)
+  /* the set_state virtual function was renamed to change_state in glib 2.29.90 */
+#if GLIB_CHECK_VERSION (2, 29, 90)
   iface->change_state = mx_action_set_state;
 #else
   iface->set_state = mx_action_set_state;
-- 
1.7.6.2

